### ISSUE TITLE
tests.foreman.cli.test_variables.test_negative_create_empty_matcher_value - smart variable not found

### ISSUE THREAD

**Issue by rplevka** [SatelliteQE/robottelo#4111]
there's possible (regression) bug:

```bash
2016-12-19 11:20:19 - robottelo.ssh - DEBUG - >>> LANG=en_US.UTF-8  hammer -v -u admin -p changeme --output=csv smart-variable create --default-value="20" --variable="EPlWR1sFpl" --puppet-class="cli_test_variables" --variable-type="integer"
2016-12-19 11:20:21 - robottelo.ssh - DEBUG - <<< stdout
Message,Id
Smart variable [EPlWR1sFpl] created,84

2016-12-19 11:20:21 - robottelo.ssh - INFO - Destroying Paramiko client 0x7f0124c530d0
2016-12-19 11:20:21 - robottelo.ssh - INFO - Destroyed Paramiko client 0x7f0124c530d0
2016-12-19 11:20:21 - robottelo.ssh - INFO - Instantiated Paramiko client 0x7f0124c11e50
2016-12-19 11:20:22 - robottelo.ssh - DEBUG - Connected to [qe-sat63-rhel7.satqe.lab.eng.rdu2.redhat.com]
2016-12-19 11:20:22 - robottelo.ssh - DEBUG - >>> LANG=en_US.UTF-8  hammer -v -u admin -p changeme --output=json smart-variable info --id="84"
2016-12-19 11:20:24 - robottelo.ssh - DEBUG - <<< stderr
[ERROR 2016-12-19 11:20:23 API] 404 Resource Not Found
[ERROR 2016-12-19 11:20:23 Exception] Smart variable not found by id '84'
Smart variable not found by id '84'
```

**Comment by svtkachenko** [comment]
Nope, just TearDown in parallel run, must be fixed with #4109

**Comment by odovzhenko** [comment]
@rplevka #4109 was addressed and this test works fine in Jenkins.
Probably we need close this issue.
